packages/ui/src/1_atoms/Paragraph/Paragraph.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { render, screen } from '@testing-library/react';
+
+import React from 'react';
+
+import { Paragraph } from './Paragraph';
+
+describe('Test paragraphs', () => {
+  test('renders a paragraph', () => {
+    render(<Paragraph children="Testing paragraph" />);
+    expect(screen.getByText('Testing paragraph')).toBeInTheDocument();
+  });
+
+  test('check paragraph instance', () => {
```

Comment:
Pls add another test with some different than the base configuration and check classnames if they are set correctly.